src/javascripts/events/crewDomEvent.js

```diff
@@ -1,22 +1,43 @@
+/* eslint-disable object-curly-newline */
```

Comment:
This needs to be removed.

Reply:
Is there another way this is supposed to be formated? @TrinityChristiana